client/src/partials/journal/journal.js

```diff
@@ -148,6 +148,6 @@ function JournalController(Journal, Sorting, Grouping, Filtering, Columns, Confi
 
   //This function opens a modal, to let the user posting transaction to the general ledger
   vm.openTrialBalanceModal = function openTrialBalanceModal () {
-    PostingService.openTrialBalanceModal(vm.grouping.getSelectedRows());
+    $state.go('trialBalanceMain', {records : vm.grouping.getSelectedGroups()}, {reload : false});
```

Comment:
By default, [reload is false](https://ui-router.github.io/docs/0.3.1/#/api/ui.router.state.$state#methods_go).  When you write `{ reload: false }` explicitly, it makes other developers think it is a mistake (because why would you set a default value?).

Please avoid setting default values explicitly in the future.